pkg/storaged/client.js

```diff
@@ -1069,11 +1069,17 @@
                                                                                    "/org/storage/stratis2",
                                                                                    { watch: false });
 
+                stratis.subscribe(
+                    {
+                        path_namespace: "/org/storage/stratis2",
+                        interface: "org.freedesktop.DBus.Properties",
+                        member: "PropertiesChanged",
+                        arg0: "org.storage.stratis2.filesystem"
+                    },
+                    (path, _iface, _signal, args) => stratis2_fixup_pool_notifications(path, args[1]));
+
                 return stratis.watch({ path_namespace: "/org/storage/stratis2" }).then(() => {
-                    client.stratis_manager.client.addEventListener('notify', (event, data) => {
-                        client.update();
-                        stratis2_fixup_pool_notifications(data);
-                    });
+                    client.stratis_manager.client.addEventListener('notify', () => client.update());
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18361-20230217-045622-6af132ab-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)